gremlin-server/src/test/scripts/generate-all.groovy

```diff
@@ -36,7 +36,7 @@ globals << [hook : [
     // things now, that test config would have been mixed in with release artifacts and there would have been ugly
     // exclusions to make packaging work properly.
     allowSetOfIdManager = { graph, idManagerFieldName, idManager ->
-        java.lang.reflect.Field idManagerField = graph.class.getDeclaredField(idManagerFieldName)
+        java.lang.reflect.Field idManagerField = graph.class.getSuperclass().getDeclaredField(idManagerFieldName)
```

Comment:
I'm not very familiar with this script, is `graph` in this context guaranteed to be an instance of either TinkerGraph or TinkerTransactionGraph? Is it possible that this may be used by other providers and inserting `getSuperclass()` will break their testing?

Reply:
yes, other providers should use own scripts